handlers/validators.go

```diff
@@ -34,33 +34,46 @@ func Validators(w http.ResponseWriter, r *http.Request) {
 	}
 
 	latestEpoch := services.LatestEpoch()
-
 	validatorOnlineThresholdSlot := GetValidatorOnlineThresholdSlot()
+
 	for _, validator := range validators {
```

Comment:
all the checks in this for-loop can be replaced by checking against `validator.Status`